docs/protocol/evmos-cli/manual-localnet.md

```diff
@@ -1,8 +1,10 @@
 ---
-sidebar_position: 2	
+sidebar_position: 2
 ---
 
-# Single Node
+# Manual Localnet
+
+<!-- TODO: Descripe steps to start a local network without a script from scratch -->
```

Comment:
[Install Binary] is the not pointing to the right link, can you fix that?

Reply:
Additionally, can you write a few lines around manual localnet so it help sets the context?

Reply:
Actually, re-reading this, this is section is a bit confusing, since we both have automated and manual, the title does not fully match. I think its better to be clear and call it Localnet and then use the title header to explain each section.

Reply:
As described in the description of this PR, these files don't need to be reviewed.